main_app/templates/main_app/register.html

```diff
@@ -67,24 +39,87 @@ <h1 class="display-4" style="text-align: center; color: black; font-family: 'Ban
                 }
             }
 
-            @media screen and (min-device-width: 425px) and (max-device-width: 768px) {
-                body {
-                    font-size: 1.5rem;
-                }
-
-                label {
-                    font-size: 1.5rem;
-                }
-            }
-
             @media only screen and (min-width:768px) {
                 .register-page {
                     position: relative;
                 }
             }
+            /* Register Page styling */
+            .register-page{
+                margin-top: 8em;
+                margin-bottom: 4em;
+            }
+            .register-page-img {
+                width: 35%;
+                height: 35%;
+                display: block;
+                margin: 0 auto;
+                margin-bottom: 0.5em;
+            }
+
+            .register-page-title {
+                font-size: 2.75rem;
+                font-family: sans-serif;
+                font-weight: 900;
+                text-align: center;
+                margin-bottom: 0.5em;
+            }
+
+            .register-page-text {
+                font-size: 2rem;
+                text-align: center;
+            }
+
+            .form-style {
+                font-size: 1.15rem;
```

Comment:
Reduce the width of the form. Make it `width: 60%;`.

Reply:
If I make it 60% then I cannot change it for smaller screen because 60% is looking empty for smaller screens

Reply:
Can you please reduce the width only for bigger screens?

Reply:
Done. Please check. If it's not working let me know